applications/openshift/ocp-permissions/ocp-files/file_permissions_master_openvswitch/rule.yml

```diff
@@ -30,5 +30,7 @@ template:
     name: file_permissions
     vars:
         filepath: /etc/origin/openvswitch/
-        file_regex: ^[^\.].*$
+        file_regex: ^[^.].*$
         filemode: '0644'
+    backends:
+        bash: 'off'
```

Comment:
@rmetrich Unfortunately the current file permissions bash remediation template doesn't support this `file_regex` as @yuumasato mentioned above. The bash remediation applies the regex to the full path, instead of only the base file name. Ansible remediation should be fine. Do you need the bash remediation? Or can we disable for the moment?

Reply:
@ggbecker I guess my customers use Ansible remediation mostly, I will update here if I know more.